celery/backends/database/__init__.py

```diff
@@ -113,6 +113,13 @@ def _store_result(self, task_id, result, state,
             task.result = result
             task.status = state
             task.traceback = traceback
+            if self.app.conf.find_value_for_key('extended', 'result'):
+                task.name = task
+                task.args = args
```

Comment:
since args and kwargs are tuples, dicts, maybe we should consider storing them in JSON?

Reply:
my source is https://github.com/celery/celery/blob/master/celery/backends/base.py#L701

Reply:
(or maybe using the result serializer `self.serializer` to encode the args/kwargs)

Reply:
Btw: I can help if you don't have time.

Reply:
@adimux I would happy accept some help!

Reply:
@adimux added you as a collab on my fork: https://github.com/mrkcmo/celery/invitations

Reply:
great, working on it

Reply:
@adimux  any updates?

Reply:
I couldn't work on it a lot the last weeks, but this week I'll probably have a working version.

Reply:
Resolved by @adimux